src/script/zcash_script.h

```diff
@@ -63,6 +63,26 @@ void* zcash_script_new_precomputed_tx(
     unsigned int txToLen,
     zcash_script_error* err);
 
+/// Deserializes the given transaction and precomputes values to improve
+/// script verification performance. Must be used for V5 transactions;
+/// may also be used for previous versions.
+///
+/// allPrevOutputs must point to the concatenation of all outputs from previous
+/// transactions that are spent by the inputs of the given transaction.
+/// The outputs must be encoded as specified by Bitcoin (value followed by
+/// pk_script, including a leading CompactSize).
```

Comment:
This reads weirdly; it suggests the format is `(CompactSize, value, pk_script)` when it is actually `(value, CompactSize, pk_script)`.

Reply:
Agreed, reworded with the encoding change in b459f197f